### PR TITLE
add util-linux package related dependencies libcap-ng, libecon…

### PR DESCRIPTION
**Pull request by rawlingsj**
…f and linux-pam

Signed-off-by: James Rawlings <jrawlings@chainguard.dev>

### CODE REVIEW

**Comment by rawlingsj** [comment]
Oops forgot `gettext` and `libunistring`.  Adding them shortly...

**Review by kaniini — DISMISSED** [review]

File: Makefile [hunk 1]
```diff
@@ -245,5 +245,12 @@ $(eval $(call build-package,php,8.1.13-r0))
 $(eval $(call build-package,composer,2.4.4-r0))
 $(eval $(call build-package,docker-credential-ecr-login,0.6.0-r1))
 $(eval $(call build-package,wait-for-it,0.20200823-r1))
+$(eval $(call build-package,libunistring,1.1-r0))
+$(eval $(call build-package,gettext,0.21.1-r0))
+$(eval $(call build-package,utmps,0.1.2.0-r0))
```
Comment (kaniini) [inline Makefile:250]: utmps is not needed on wolfi, as glibc supports utmp.

File: util-linux/rfkill.initd [hunk 2]
```diff
@@ -0,0 +1,50 @@
+#!/sbin/openrc-run
```
Comment (kaniini) [inline util-linux/rfkill.initd:1]: wolfi does not use openrc

**Review by luhring — DISMISSED** [review]
Looks good, pending resolution of @kaniini 's comments

File: Makefile (hunk 1 above)
Comment (rawlingsj) [reply Makefile:250]: ok great - lemme remove utmps

**Review by jdolitsky — APPROVED** [review]

**Review by luhring — APPROVED** [review]

File: util-linux.yaml [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+  epoch: 0
+  description: Random collection of Linux utilities
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: |-
+        GPL-3.0-or-later AND GPL-2.0-or-later AND GPL-2.0-only AND PL-1.0-only AND LGPL-2.1-or-later AND LGPL-1.0-only AND BSD-1-Clause AND BSD-3-Clause AND BSD-4-Clause-UC AND MIT AND Public-Domain
```
Comment (mattmoor) [inline util-linux.yaml:13]: @rawlingsj Is the `PL-1.0-only` here meant to be `GPL-1.0-only`?  This is failing the NTIA license check as an invalid SPDX license

**Pull request merged**